Command/ValidateConnectionsCommand.php

```diff
@@ -107,11 +107,19 @@ private function getRow(ConnectionInterface $connection): array
             $result = sprintf('<error>FAILED: %s</error>', $exception->getErrors('last'));
         }
 
+        if($connection->isEnabled())
```

Comment:
Please add space between if and "(" (see: Symfony Coding Standard https://symfony.com/doc/current/contributing/code/standards.html#symfony-coding-standards-in-detail)